javascript/package.json

```diff
@@ -21,7 +21,7 @@
     "lint": "lerna run lint --stream"
   },
   "devDependencies": {
-    "lerna": "^3.16.4"
+    "lerna": "^3.22.1"
```

Comment:
Were there any problems with 3.16.4?

Reply:
No, I'm not sure why the version is upgraded tbh.